command.go

```diff
@@ -371,6 +372,14 @@ func (cmd *SliceCmd) String() string {
 	return cmdString(cmd, cmd.val)
 }
 
+// Scan scans the results from the map into a destination struct. The map keys
+// are matched in the Redis struct fields by the `redis:"field"` tag.
+func (cmd *SliceCmd) Scan(dest interface{}) error {
+	// Pass the list of keys and values. Skip the first to args (command, key),
+	// eg: HMGET map.
+	return hscan.Scan(cmd.args[2:], cmd.val, dest)
```

Comment:
@knadh here you ignore cmd.err (and *StringStringMapCmd), 
mget command will encounter an error, it should run normally, here is: cmd.args[?:]
mget: mget key1 key2 key3 ...
hmget hmget key field1 field2 field3

Reply:
Good spot @monkey92t. Fixed here https://github.com/knadh/redis/commit/077e145f6a26a8a441fa45d2dadb6526935beca3